lib/index.js

```diff
@@ -52,7 +57,7 @@ internals.purdy = function (object, options) {
 module.exports = function (object, options) {
 
     var Purdy = new internals.purdy(object, options);
-    return console.log(Purdy.stringify());
+    return consoleLog(Purdy.stringify());
```

Comment:
@danielb2 is this not the only place where Purdy calls `console.log`? Does it call `console.log` via dependencies, as well?